### DISCUSSION TITLE
Is there any step by step tutorial on how to run Pocketbase on Google Cloud Platform i.e. Compute Engine?

### DISCUSSION THREAD

**Discussion by Mashiane** [pocketbase/pocketbase#2617] [Q&A]
Hi

I need some help running PB on GCP compute engine. Can someone please provide a step by step on how to do that?
Thank you so much.

**Comment by PSOwned** [comment]
Also trying to run pocketbase in gcloud and gcloud logs say: 
terminated: Application failed to start: Failed to create init process: failed to load /usr/local/bin/pocketbase: permission denied

Someone have a solution for this?
